packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <div id="oc-file-details-sidebar">
+    <oc-loader v-if="loading" />
+    <div v-else>
+      <div
+        v-if="highlightedFile.preview"
+        class="details-preview uk-flex uk-flex-middle uk-flex-center oc-mb-m"
+      >
+        <oc-img :src="highlightedFile.preview" alt="" />
+      </div>
+      <div v-if="showShares" data-test-id="sharingInfo" class="uk-flex uk-flex-middle oc-my-m">
+        <oc-button
+          v-if="hasPeopleShares"
+          v-oc-tooltip="peopleSharesLabel"
+          appearance="raw"
+          class="oc-mr-xs"
+          :aria-label="peopleSharesLabel"
+          @click="expandPeoplesAccordion"
+        >
+          <oc-icon name="group" />
+        </oc-button>
+        <oc-button
+          v-if="hasLinkShares"
+          v-oc-tooltip="linkSharesLabel"
+          appearance="raw"
+          class="oc-mr-xs"
+          :aria-label="linkSharesLabel"
+          @click="expandLinksAccordion"
+        >
+          <oc-icon name="link" />
+        </oc-button>
+        <p class="oc-my-rm oc-mx-s" v-text="detailSharingInformation" />
+      </div>
+      <table class="details-table" aria-label="detailsTableLabel">
+        <tr v-if="hasTimestamp" data-test-id="timestamp">
+          <th scope="col" class="oc-pr-s" v-text="timestampTitle" />
+          <td>
+            <oc-button
+              v-if="showVersions"
+              v-oc-tooltip="seeVersionsLabel"
+              appearance="raw"
+              :aria-label="seeVersionsLabel"
+              @click="expandVersionsAccordion"
+              v-text="capitalizedTimestamp"
+            />
+            <span v-else v-text="capitalizedTimestamp" />
+          </td>
+        </tr>
+        <tr v-if="ownerName" data-test-id="ownerName">
+          <th scope="col" class="oc-pr-s" v-text="ownerTitle" />
+          <td>
+            <p class="oc-m-rm">
+              {{ ownerName }}
+              <span v-if="ownedByCurrentUser" v-translate>(me)</span>
+            </p>
+          </td>
+        </tr>
+        <tr data-test-id="sizeInfo">
```

Comment:
have we decided to use data-test-id or data-testid?

Reply:
The latter (data-testid) but haven't done search&replace in `web` yet

Reply:
I did the search & replace in the ODS update PR. Could you switch to `data-testid` in your PR as well?

Reply:
Surely, thanks 🙌🏽